numpy/random/mtrand/mtrand.pyx

```diff
@@ -4198,12 +4198,12 @@ cdef class RandomState:
         -----
         The probability density for the Hypergeometric distribution is
 
-        .. math:: P(x) = \\frac{\\binom{m}{n}\\binom{N-m}{n-x}}{\\binom{N}{n}},
+        .. math:: P(x) = \\frac{\\binom{m}{x}\\binom{N-m}{n-x}}{\\binom{N}{n}},
```

Comment:
Perhaps it would be good to just use `ngood` and `nbad` directly here?

Reply:
Hi, do you suggest directly putting `\text{ngood}`into the equation? Wouldn't that be a little bit verbose? I personally think it's OK to use single-letter variables for math, but if you do prefer the explicit way, I will revise it.

Reply:
I more object to the fact that the letters don't even correspond to the variables here, but to combinations of them.

`n_g` and `n_b` would be fine here too

Reply:
I see your concern, maybe we should follow the convention of `Distributions.jl`'s [documentation](https://juliastats.github.io/Distributions.jl/latest/univariate.html#Distributions.Hypergeometric), but using `g` and `b` instead of `s` and `f` there? Putting `N` in the equation makes more sense if the API were designed similar to boost's [`hypergeometric_distribution`](https://www.boost.org/doc/libs/1_52_0/libs/math/doc/sf_and_dist/html/math_toolkit/dist/dist_ref/dists/hypergeometric_dist.html).